packages/components/src/components/hds/app-footer/status-link.ts

```diff
@@ -5,30 +5,32 @@
 
 import Component from '@glimmer/component';
 import { htmlSafe } from '@ember/template';
+import type { SafeString } from '@ember/template/-private/handlebars';
 import { assert } from '@ember/debug';
 
-export const STATUSES = {
-  operational: {
-    text: 'System operational',
-    iconName: 'check-circle',
-  },
-  degraded: {
-    text: 'System degraded',
-    iconName: 'alert-triangle',
-  },
-  maintenance: {
-    text: 'System maintenance',
-    iconName: 'alert-triangle',
-  },
-  outage: {
-    text: 'System outage',
-    iconName: 'x-circle',
-  },
-};
+import type { HdsInteractiveSignature } from '../interactive/';
+import { HdsAppFooterStatusLinkStatusValues } from './types.ts';
+import type { HdsAppFooterStatusTypes } from './types.ts';
 
-export default class HdsAppFooterStatusLinkComponent extends Component {
-  constructor() {
-    super(...arguments);
+export const STATUSES = HdsAppFooterStatusLinkStatusValues;
+
+export interface HdsAppFooterStatusLinkSignature {
+  Args: HdsInteractiveSignature['Args'] & {
+    itemStyle?: SafeString;
+    status?: HdsAppFooterStatusTypes;
+    statusIcon?: string;
+    statusIconColor?: string;
+    text?: string;
+  };
+  Blocks: {
+    default: [];
+  };
+  Element: HdsInteractiveSignature['Element'];
```

Comment:
```suggestion
  Element: HdsAppFooterLinkSignature['Element'];
```
(remember to import `HdsAppFooterLinkSignature`)